test/functional/api/arbmark_spec.lua

```diff
@@ -0,0 +1,18 @@
+-- Sanity checks for buffer_* API calls via msgpack-rpc
```

Comment:
I have no idea what is supposed to be placed into functional/viml. dict_notifications_spec should be probably in eval. viml/function_spec should be in eval (and I have no idea what `api_info` is doing in this file, I would place it separately because `function_spec` is too generic; this is fine if you are testing generic function-related features like MAX_FUNC_ARGS handling, but this is no place for testing specific functions), as well as viml/errorlist_spec: other functions are tested there. Not sure about viml/lang_spec and viml/completion_spec: first may be in eval, second is such a feature that can have its own directory.